### DISCUSSION TITLE
Bug: unable to add multiline comment to "partial" diffs

### DISCUSSION THREAD

**Discussion by dimas** [community/community#74460] [Pull Requests]
### Select Topic Area

Question

### Body

While I can successfully create multi-line comments in a PR when looking at "Show all changes" by clicking on one line and dragging the cursor over multiple lines, this feature stopped working when I am looking at "partial" diff. What I mean by "partial" diff is when I am looking at "Show changes since your last review". Github shows that plus icon like normal and I can drag it and in the end even even that panel is shown:

![image](https://github.com/community/community/assets/93186/38ee6a9d-3187-4d4a-b727-9af7363a1a90)

but there is no text input to enter the comment.
And JS console shows this:

![image](https://github.com/community/community/assets/93186/3042b3fa-e1c7-4b07-8cf7-bfe02bc557cf)

(No idea if it is related or not though)

**Comment by Rohland** [comment]
Was about to log this as well. I've tried Safari, Chrome and Firefox. All experiencing the same issue highlighted above.

**Reply by dimas** [reply]
JFYI, I logged it as a support ticket as well, in addition to this discussion. I am told devs were able to reproduce,accepted it as a bug and working on it.
